public/gbm_cptac_2021/meta_microRNA_quantification.txt

```diff
@@ -0,0 +1,10 @@
+genetic_alteration_type: GENERIC_ASSAY
+show_profile_in_analysis_tab: true
+data_filename: data_microRNA_quantification.txt
+datatype: LIMIT-VALUE
+stable_id: microRNA
+profile_name: miRNA expression (TPM)
+profile_description: Expression of micro RNA abundance in TPM values
+generic_assay_type: Acetylproteome
```

Comment:
This does not seem correct. As a result the miRNA data is placed under the header acetylproteome.

Besides, there is a way to load miRNA expression as actual "expression" data using the cBioPortal format:
https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#expression-data